bin/asset-pipeline

```diff
@@ -256,7 +260,20 @@ class AssetPipeline {
       )
       .process(text);
 
-    const fileHash = this.getContentHash(output.html);
+    let { html } = output;
+
+    // Replace the default social image in `meta` tag, which isn't supported by
+    // posthtmlURL.
```

Comment:
posthtml-urls? Might be good to file and reference an issue: https://github.com/posthtml/posthtml-urls/issues so we can track if that gets fixed.